client/src/components/Home.tsx

```diff
@@ -0,0 +1,188 @@
+
+  return (
+    <>
+      <NavHeader logout={props.handleLogout} login={props.login} />
+
+      <div className="body-container">
+        {/*<MapComponent apiKey={""} />*/}
+        {
+          <table className="table-documents">
+            <thead>
+              <tr>
+                <th>Icon</th>
+                <th>Title</th>
+                <th>Info</th>
+              </tr>
+            </thead>
+            <tbody>
+              {documents.map((document) => (
+                <tr key={document.id}>
+                  <td>
+                    <img
+                      className="doc-icon"
+                      src="/document-icon.png"
+                      alt="Document icon"
+                    />
+                  </td>
+                  <td className="doc-title">{document.title}</td>
+                  <td>
+                    <button
+                      className="icon-info"
+                      onClick={() => {
+                        console.log(documents);
+                        setSidebarOpen(true);
+                        setDocSelected(document);
+                      }}
+                    >
+                      Info
+                    </button>
+                  </td>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+        }
+
+        <div className={`sidebar ${sidebarOpen ? "open" : ""}`}>
+          {<Sidebar setSidebarOpen={setSidebarOpen} document={docSelected} />}
+        </div>
+      </div>
+
+      {/* Button to add a new Document (Description) */}
+      <div>
+        <button className="add-document" onClick={handleAddButton}>
+          Add new Document
+        </button>
+      </div>
+
+      {/* Modal Add Component */}
+      <ModalAdd
+        modalOpen={modalOpen}
+        onClose={() => setModalOpen(false)}
+        onSubmit={handleAddDocument}
+        documents={documents}
+      />
+
+      {/* <div className="diagram">
+                Diagram part
+            </div> */}
+    </>
+  );
+};
+
+function Sidebar(props: {
+  setSidebarOpen: (isOpen: boolean) => void;
+  document: Document | null;
+}) {
+  return (
+    <>
+      <div className="container-btns">
+        <button
+          className="btn-download-sidebar"
+          onClick={() => props.setSidebarOpen(false)}
+        >
+          <img
+            className="btn-download-img"
+            src="/file-earmark-arrow-down.svg"
+            alt="Download"
+          />
+        </button>
+        <button
+          className="btn-close-sidebar"
+          onClick={() => props.setSidebarOpen(false)}
+        >
+          <img className="btn-close-img" src="/x.svg" alt="Close" />
+        </button>
+      </div>
+      <div className="content">
+        <img src="/icons8-under-construction-50.png" alt="Under Construction" />
+        <hr />
+        <h3>{props.document?.title}</h3>
+        <p>{props.document?.description}</p>
+        <hr className="separator" />
+        <h6>
```

Comment:
h6 should be nested inside h4 to have semantic HTML. Use css for font-size instead

Reply:
Ok, makes sense